### DISCUSSION TITLE
gif image in table of markdown file clipping through the top bar

### DISCUSSION THREAD

**Discussion by laggykiller** [community/community#83639] [Repositories]
### Select Topic Area

Bug

### Body

gif image in table of markdown file clipping through the top bar. Does not occur for png file. Does not occur if not inside table.

Seems to be related to https://github.com/orgs/community/discussions/17684

If 'Autoplay animated images' is enabled then the bug does not occurs, if disabled then the bug occurs.

You can see the bug here:
https://github.com/laggykiller/lihkg-icons/blob/main/view/normal.md

<img width="469" alt="image" src="https://github.com/community/community/assets/61652821/c0c6e17e-59ff-4be5-97fd-980f0ca55837">

**Comment by Kaparrs** [comment]
Not having any clipping issues

**Reply by laggykiller** [reply]
What browser are you using? I have tested with latest firefox, chrome and brave browser.

You have to scroll down to see the bug

**Reply by Kaparrs** [reply]
I'm using Brave, I scrolled down but still looks normal

![image](https://github.com/community/community/assets/71441748/21962f00-a48f-43ff-b2fa-7abffb177451)

**Reply by laggykiller** [reply]
Go to `brave://version`

My Brave version is `1.61.109 Chromium: 120.0.6099.144 (Official Build) (64-bit) `

I am running on Windows 11 Version 23H2.

Tested on two different machines, one with Nvidia GPU, another with Intel iGPU, both facing same problem.

**Reply by laggykiller** [reply]
Tested on even more machines

Same bug on my Android phone (Brave browser, rotated screen) and macOS arm64 Sonoma (Brave browser).

The bug does not appear on Ubuntu 20.04 live environment (Firefox browser) and Manjaro live environment (Midori browser)

**Reply by Kaparrs** [reply]
> Go to `brave://version`
> 
> 
> 
> My Brave version is `1.61.109 Chromium: 120.0.6099.144 (Official Build) (64-bit) `
> 
> 
> 
> I am running on Windows 11 Version 23H2.
> 
> 
> 
> Tested on two different machines, one with Nvidia GPU, another with Intel iGPU, both facing same problem.

Same version. I got Windows 10 tho

**Reply by laggykiller** [reply]
Interesting, I have tested on Windows 10 using qemu VM with qxl graphics driver. I can reproduce with Chrome and Brave browser.

Hardware of your machine, esp. GPU (?)

**Reply by laggykiller** [reply]
Wait, I found out why...

See https://github.com/orgs/community/discussions/17684

Go to https://github.com/settings/accessibility

If 'Autoplay animated images' is enabled then the bug does not occurs, if disabled then the bug occurs.

**Comment by soul-xx** [comment]
nice
